MantidQt/API/src/MantidQwtWorkspaceData.cpp

```diff
@@ -27,53 +38,64 @@ operator=(const MantidQwtWorkspaceData &data) {
 /// Calculate absolute minimum and maximum values in a vector. Also find the
 /// smallest positive value.
 void MantidQwtWorkspaceData::calculateYMinAndMax() const {
-
-  const double maxDouble = std::numeric_limits<double>::max();
-  double curMin = maxDouble;
-  double curMinPos = maxDouble;
-  double curMax = -maxDouble;
+  // Set this to true to get the "real" data size
+  // It's correct value is then recalculated below. This is not
+  // too nice but a big refactor is not worth it given the new
+  // workbench/plotting developments.
+  m_plottable = DataStatus::Plottable;
+  m_minY = m_maxY = m_minPositive = 0.0;
```

Comment:
Is this for initialisation only? `m_minPositive` cannot be 0 for plottable data.